src/planner/binder/statement/bind_create.cpp

```diff
@@ -172,7 +212,7 @@ BoundStatement Binder::Bind(CreateStatement &stmt) {
 		break;
 	}
 	case CatalogType::MACRO_ENTRY: {
-		auto schema = BindCreateFunctionInfo(*stmt.info);
+		auto schema= BindCreateFunctionInfo(*stmt.info);
```

Comment:
Did you run the formatter before submitting this PR (`make format-fix`)?

Reply:
will run the formatter